<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>437</width>
    <height>292</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>437</width>
    <height>292</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Avenir Next</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Авторизация в систему</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgba(230, 230, 250, 255), stop:0.5 rgba(135, 206, 235, 255), stop:1 rgba(70, 130, 180, 255));
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}

#pushButton:hover {
background-color: #9db1cc
}</string>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2 {
background-color: #b0c4de;
border-radius:7%;
border: 1px solid grey;
font-family: Avenir Next;
font-size: 16px;
}
#pushButton_2:hover {
background-color: #9db1cc
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label {
font-family: Avenir Next;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir Next</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2 {
font-family: Avenir Next;
font-size: 16px;
}</string>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit {
border-radius:5%;
background-color: #e0f0ff;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lineEdit_2 {
border-radius:5%;
background-color: #e0f0ff;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
